code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -620,25 +621,28 @@ Code:
 			active1 = find_record("id", href_list["target"], data_core.general)
 			if(active1)
 				active3 = find_record("id", href_list["target"], data_core.security)
-			loc:mode = 451
+			pda.mode = 451
 			mode = 451
 			if(!active3)
 				active1 = null
 
 		if("Send Signal")
 			spawn( 0 )
-				radio:send_signal("ACTIVATE")
+				var/obj/item/radio/integrated/signal/S
+				S.send_signal("ACTIVATE")

```

Comment:
it ********loooks********* like it is type casted properly, but its not, those procs/vars only exist for /obj/item/radio/integrated/signal not /obj/item/radio/integrated that radio is typecasted to, and i can't re-typecast it, because pda's only get a /obj/item/radio/integrated/signal on rd and science and generic remote singler cartridges,